src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -39,20 +59,20 @@
 <ng-template #readOnlyFeedback>
     <div class="container">
         <div class="row align-items-center">
-            <div class="col ps-0">
+            <div class="col ps-0 pt-1">
                 <h5 class="d-inline">
                     <span class="badge" [class.bg-success]="feedback.credits! > 0" [class.bg-danger]="feedback.credits! < 0" [class.bg-warning]="feedback.credits === 0">{{
                         roundScoreSpecifiedByCourseSettings(feedback.credits, course) + 'P'
                     }}</span>
                 </h5>
                 <h6 class="d-inline" jhiTranslate="artemisApp.assessment.detail.tutorComment">Tutor Comment:</h6>
             </div>
-            <div *ngIf="!readOnly" class="col d-flex justify-content-end">
+            <div *ngIf="!readOnly" class="col d-flex justify-content-end pt-1">
                 <button type="submit" (click)="editFeedback(codeLine)" class="btn btn-primary btn-sm">
                     <fa-icon [icon]="'pencil-alt'"></fa-icon> <span jhiTranslate="entity.action.edit">Edit</span>
                 </button>
             </div>
         </div>
-        <div class="row ms-4 mt-2">{{ feedback.detailText }}</div>
+        <div class="row ms-4 mt-2"><span [innerHTML]="buildFeedbackTextForCodeEditor(feedback)"></span></div>
```

Comment:
```suggestion
        <div class="row ms-4 mt-2 pb-1"><span [innerHTML]="buildFeedbackTextForCodeEditor(feedback)"></span></div>
```
to have the same spacigng bottom as top